features/fixtures/ios-swift-cocoapods/iOSTestApp/scenarios/OOMSessionScenario.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  OOMLoadScenario.swift
+//  iOSTestApp
+//
+//  Created by Alexander Moinet on 13/10/2020.
+//  Copyright © 2020 Bugsnag. All rights reserved.
+//
+
+import Foundation
+import Bugsnag
+
+class OOMSessionScenario: Scenario {
+
+    override func startBugsnag() {
+        config.enabledErrorTypes.ooms = true
+        config.autoTrackSessions = true
+        Bugsnag.start(with: config)
+    }
+
+    override func run() {
+//        Bugsnag.startSession()
```

Comment:
Presumably this should be removed?

Reply:
Ah, that bit is correct, the `autoTrackSessions` opt needed to be changed